src/plays/simple-live-chat/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Simple Live Chat
```

Comment:
Put some more info in the readme. consider this like the content someone will read to understand this work.

Reply:
yes added